src/utils/components/Consoles/components/CloudInitCredentials/CloudInitCredentials.tsx

```diff
@@ -2,26 +2,29 @@ import React, { FC } from 'react';
 
 import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
 import HelpTextIcon from '@kubevirt-utils/components/HelpTextIcon/HelpTextIcon';
+import useK8sWatchData from '@kubevirt-utils/hooks/useK8sWatchData';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { VirtualMachineModelGroupVersionKind } from '@kubevirt-utils/models';
-import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
 import { Flex, Skeleton, Title } from '@patternfly/react-core';
 
 import CloudInitCredentialsContent from './CloudInitCredentialsContent';
 
 import './cloud-init-credentials.scss';
 
 type CloudInitCredentialsProps = {
+  vmCluster?: string;
   vmName: string;
   vmNamespace: string;
 };
 
 const CloudInitCredentials: FC<CloudInitCredentialsProps> = ({
+  vmCluster,
   vmName: name,
   vmNamespace: namespace,
 }) => {
   const { t } = useKubevirtTranslation();
-  const [vm, isLoaded] = useK8sWatchResource<V1VirtualMachine>({
+  const [vm, isLoaded] = useK8sWatchData<V1VirtualMachine>({
```

Comment:
We could also create a wrapper hook for `useK8sWatchData` instead of passing in the cluster.